SettingsList.py

```diff
@@ -2108,22 +2108,37 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             'randomize_key': 'randomize_settings',
         },
     ),
-    Checkbutton(
+    Combobox(
```

Comment:
I think it would be a good idea to move the setting above "shuffle cows" in `settings_mapping.json`, so it doesn't end up being a dropdown in the middle of checkboxes. Maybe it should even be at the top, before shopsanity, since it's arguably the most important "shuffle" setting.